tests/assertions/internal-libraries.n

```diff
@@ -169,4 +169,7 @@ let pub main = [_: ()] -> cmd[()] {
 		onDisconnect: websocket.user -> maybe[{ code: int; reason: str }] -> cmd[bool]
 	}
 	assert type websocket.createServer : setupOptions -> int -> cmd[maybe[()]]
+	assert value 1 == 2
```

Comment:
Won't this fail?

Reply:
oh that was a check to make sure that that stuff was being run, odd that it did not fail.